ScratchWikiSkinTemplate.php

```diff
@@ -125,6 +125,7 @@ public function execute() {
 		</ul>
 	</div>
 </div>
+<div id="not-nav">
```

Comment:
Do you think it's worth indenting the children here? I know it would be an obnoxious change but may help with readability.